cuizinart/main.py

```diff
@@ -38,6 +39,14 @@ def parse_json(obj):
     return backend, product, geojson, start_time, end_time, request_variables, horizons, issues
 
 
+def check_points_in_shape(shp, pts):
```

Comment:
This will not match a shape that is entirely within the bounds specified by the points, right?

Reply:
We include the product if they have intersections.